examples/todos-app/src-tauri/build.rs

```diff
@@ -1,3 +1,4 @@
 fn main() {
+  println!("cargo:rustc-env=MACOSX_DEPLOYMENT_TARGET=10.13");
```

Comment:
Added work-around to allow binaries to work in a larger number of macOS versions